Index: rules
===================================================================
--- rules	(revision 2445)
+++ rules	(working copy)
@@ -31,6 +31,9 @@
 	      -DMAN_INSTALL_DIR=/usr/share/man \
 	      -DINFO_INSTALL_DIR=/usr/share/info \
 	      -DDATADIR=/usr/share \
+	      -DWITH_GDA3:BOOL=OFF \
+	      -DWITH_SPEECH:BOOL=OFF \
+	      -DWITH_MAPNIK:BOOL=OFF \
 	      .. 
 
 	rm -f build/build-stamp
Index: control
===================================================================
--- control	(revision 2518)
+++ control	(working copy)
@@ -15,21 +15,14 @@
 	libgps-dev,
 	libgtk2.0-dev,
 	libltdl7-dev | libltdl3-dev,
-	libmapnik-dev (>> 0.5.0-1),
 	libpcre3-dev,
 	libpq-dev,
-	libspeechd-dev (> 0.6.6),
 	libsqlite3-dev,
 	libxml-twig-perl,
 	libxml2-dev,
 	perl-doc,
 	pkg-config,
-	po-debconf,
-	python-dev,
-	python-central,
-	python-dev | python2.4-dev | python2.3-dev,
-	python-support,
-	speech-dispatcher (>= 0.6.6)
+	po-debconf
 Standards-Version: 3.8.1
 
 # ------------------------------------------------------------------
@@ -41,22 +34,12 @@
 	libltdl7 | libltdl3,
 	libboost-filesystem1.34.1 | libboost-filesystem1.33.1,
 	libboost-thread1.34.1 | libboost-thread1.33.1,
-	libmapnik (>> 0.5.0-1) | libmapnik0.5,
 	openstreetmap-map-icons (>> 8310),
 	libsqlite3-0
 Recommends: gpsd,
-	espeak,
 	gpsdrive-data-maps (>> 2322),
-	openstreetmap-utils (>> 8290) | gpsdrive-utils (>> 2.10svn2299),
-	openstreetmap-mapnik-world-boundaries (>> 7749),
-	mapnik-plugins,
-	mapnik-utils,
-	postgresql-8.3-postgis | postgresql-8.2-postgis,
-	openstreetmap-mapnik-data,
-	ttf-dejavu,
-	python-mapnik
-Conflicts: osm-data-mapnik,
-	osm-data-map-icons,
+	openstreetmap-utils (>> 8290) | gpsdrive-utils (>> 2.10svn2299)
+Conflicts: osm-data-map-icons,
 	gpsdrive-scripts
 Suggests: grass,
 	libarchive-zip-perl,
@@ -64,11 +47,9 @@
 	libfile-find-rule-perl,
 	libgtk2.0-0,
 	libdbd-sqlite3-perl,
-	libgeos-c1,
 	sudo,
 	gdal-bin,
-	gpsbabel,
-	festival
+	gpsbabel
 Description: GPS navigation system
  GpsDrive is a car (bike, ship, plane, foot) navigation system.
  GpsDrive displays your position provided from a GPS on a zoomable map,
@@ -78,9 +59,6 @@
  .
  You will most likely want to install the gpsd package so that you
  can talk to your GPS; without that GpsDrive is just a map viewer.
- Speech output is supported if the "espeak" software is installed.
- To use maps from the OpenStreetMap project you need to install the
- openstreetmap-* packages as well.
 
 # ------------------------------------------------------------------
 Package: gpsdrive-utils
@@ -88,11 +66,6 @@
 Depends: ${misc:Depends},
 	 ${shlibs:Depends},
          ${perl:Depends},
-	 graphicsmagick-imagemagick-compat | imagemagick,
-	 python,
-	 python-support,
-	 python-central,
-	 python-dev,
 	 perl-base (>> 5.6),
 	 libtext-query-perl,
 	 libtimedate-perl,
