Index: rules
===================================================================
--- rules	(revision 2447)
+++ rules	(working copy)
@@ -33,1 +33,4 @@
 	      -DDATADIR=/usr/share \
+	      -DWITH_GDA3:BOOL=OFF \
+	      -DWITH_SPEECH:BOOL=OFF \
+	      -DWITH_MAPNIK:BOOL=OFF \
Index: control
===================================================================
--- control	(revision 2447)
+++ control	(working copy)
@@ -19,1 +19,0 @@
-	libmapnik-dev (>> 0.5.0-1),
@@ -22,1 +22,0 @@
-	libspeechd-dev (>= 0.6.7) | libspeechd-dev (>= 0.6.7~rc1-1),
@@ -28,5 +28,0 @@
-	python-dev,
-	python-central,
-	python-dev | python2.4-dev | python2.3-dev,
-	python-support,
-	speech-dispatcher (>= 0.6.6)
@@ -44,1 +44,0 @@
-	libmapnik (>> 0.5.0-1) | libmapnik0.5,
@@ -46,2 +46,0 @@
-	mapnik-plugins,
-	mapnik-utils,
@@ -55,1 +5,0 @@
-	espeak,
@@ -60,1 +60,0 @@
-	libdbd-mysql-perl,
@@ -72,1 +57,0 @@
-	festival
@@ -83,3 +67,0 @@
- Speech output is supported if the "espeak" software is installed.
- To use maps from the OpenStreetMap project you need to install the
- openstreetmap-* packages as well.
@@ -93,4 +74,0 @@
-	 python,
-	 python-support,
-	 python-central,
-	 python-dev,
