Index: rules
===================================================================
--- rules	(revision 2445)
+++ rules	(working copy)
@@ -33,1 +33,5 @@
 	      -DDATADIR=/usr/share \
+	      -DWITH_GDA3:BOOL=OFF \
+	      -DWITH_SPEECH:BOOL=OFF \
+	      -DWITH_MAPNIK:BOOL=OFF \
+	      -DLIBGPS_OLD:BOOL=ON \
Index: control
===================================================================
--- control	(revision 2517)
+++ control	(working copy)
@@ -14,1 +14,1 @@
-	libgps-dev (>= 2.90),
+	libgps-dev (>= 2.30),
@@ -16,1 +16,0 @@
-	libmapnik-dev (>> 0.5.0-1),
@@ -18,2 +18,1 @@
	libpq-dev,
-	libspeechd-dev (>> 0.6.6),
@@ -24,6 +24,2 @@
-	po-debconf,
-	python-dev,
-	python-central,
-	python-dev | python2.4-dev,
-	python-support,
-	speech-dispatcher (>= 0.6.6)
+	python-support
+	po-debconf
@@ -40,1 +40,0 @@
-	espeak,
@@ -44,2 +44,0 @@
-	mapnik-plugins,
-	mapnik-utils,
@@ -72,4 +72,2 @@
  can talk to your GPS; without that GpsDrive is just a map viewer.
- Speech output is supported if the "espeak" software is installed.
- To use maps from the OpenStreetMap project you need to install the
- openstreetmap-* packages as well.
+ Sorry no mapnik Support, since the libmapnik-dev is too old.
@@ -84,5 +84,1 @@
 	 perlmagick,
-	 python,
-	 python-support,
-	 python-central,
-	 python-dev,
