Index: control
===================================================================
--- control	(revision 2556)
+++ control	(working copy)
@@ -39,8 +39,8 @@
 	 ${shlibs:Depends},
 	libart-2.0-2,
 	libltdl7 | libltdl3,
-	libboost-filesystem1.34.1 | libboost-filesystem1.33.1,
-	libboost-thread1.34.1 | libboost-thread1.33.1,
+	libboost-filesystem1.40.0 | libboost-filesystem1.38.0,
+	libboost-thread1.40.0 | libboost-thread1.38.1,
 	libmapnik (>> 0.5.0-1) | libmapnik0.5 | libmapnik0.6 | libmapnik0.7,
 	openstreetmap-map-icons (>> 8310),
 	libsqlite3-0
@@ -51,7 +51,7 @@
 	openstreetmap-mapnik-world-boundaries (>> 7749),
 	mapnik-plugins,
 	mapnik-utils,
-	postgresql-8.3-postgis | postgresql-8.2-postgis,
+	postgresql-8.4-postgis | postgresql-8.2-postgis,
 	openstreetmap-mapnik-data,
 	ttf-dejavu,
 	python-mapnik
