Index: control
===================================================================
--- control	(revision 2364)
+++ control	(working copy)
@@ -15,1 +15,0 @@
-	libgda3-dev,
@@ -20,1 +20,0 @@
-	libmapnik-dev (>> 0.5.0-1),
@@ -22,1 +22,0 @@
-	libspeechd-dev (>= 0.6.7),
@@ -32,1 +29,0 @@
-	speech-dispatcher (>= 0.6.6)
@@ -46,2 +45,0 @@
-	libgda3-3,
-	libgda3-postgres,
@@ -90,6 +90,5 @@
  the map file is autoselected depending of the position and preferred
- scale. Speech output is supported if the "espeak" software is 
- installed. All GPS receivers which supports NMEA protocol should
+ scale.  All GPS receivers which supports NMEA protocol should
  be usable.
  For using Maps from OpenStreetMap you need to install the openstreetmap-* 
  packages.
Index: rules
===================================================================
--- rules	(revision 2354)
+++ rules	(working copy)
@@ -29,3 +29,3 @@
-	      -DWITH_GDA3:BOOL=ON \
-	      -DWITH_SPEECH:BOOL=ON \
-	      -DWITH_MAPNIK:BOOL=ON \
+	      -DWITH_GDA3:BOOL=OFF \
+	      -DWITH_SPEECH:BOOL=OFF \
+	      -DWITH_MAPNIK:BOOL=OFF \
