Index: rules
===================================================================
--- rules	(revision 2495)
+++ rules	(working copy)
@@ -34,0 +31,3 @@
+	      -DWITH_GDA3:BOOL=OFF \
+	      -DWITH_SPEECH:BOOL=OFF \
+	      -DWITH_MAPNIK:BOOL=OFF \
===================================================================
--- control	(revision 2497)
+++ control	(working copy)
@@ -18,1 +18,0 @@
-	libmapnik-dev (>> 0.5.0-1),
@@ -21,1 +21,0 @@
-	libspeechd-dev (>= 0.6.7) | libspeechd-dev (>= 0.6.7~rc1-1),
@@ -27,1 +27,1 @@
-	po-debconf,
+	po-debconf
@@ -28,5 +27,0 @@
-	python-dev,
-	python-central,
-	python-dev | python2.4-dev | python2.3-dev,
-	python-support,
-	speech-dispatcher (>= 0.6.6)
@@ -44,1 +41,0 @@
-	libmapnik (>> 0.5.0-1) | libmapnik0.5,
@@ -48,1 +34,0 @@
-	espeak,
@@ -52,2 +34,0 @@
-	mapnik-plugins,
-	mapnik-utils,
@@ -68,7 +57,6 @@
 	sudo,
 	gdal-bin,
 	gpsbabel,
-	festival
 Description: GPS navigation system
  GpsDrive is a car (bike, ship, plane, foot) navigation system.
  GpsDrive displays your position provided from a GPS on a zoomable map,
@@ -78,9 +66,6 @@
  .
  You will most likely want to install the gpsd package so that you
  can talk to your GPS; without that GpsDrive is just a map viewer.
- Speech output is supported if the "espeak" software is installed.
- To use maps from the OpenStreetMap project you need to install the
- openstreetmap-* packages as well.
 
 # ------------------------------------------------------------------
 Package: gpsdrive-utils
@@ -89,10 +74,6 @@
 	 ${shlibs:Depends},
          ${perl:Depends},
 	 imagemagick,
-	 python,
-	 python-support,
-	 python-central,
-	 python-dev,
 	 perl-base (>> 5.6),
 	 libtext-query-perl,
 	 libtimedate-perl,
